ote/src/cljs/ote/views/service_search.cljs

```diff
@@ -142,31 +144,85 @@
      [:div.result-interfaces
       [external-interface-links e! service]]]))
 
-(defn results-listing [e! app {:keys [results empty-filters? total-service-count
-                                  filter-service-count fetching-more?]}]
-  [:div.col-xs-12.col-md-12.col-lg-12
-   [:p
-    (tr [:service-search (if empty-filters?
-                           :showing-latest-services
-                           (case filter-service-count
-                             0 :no-results
-                             1 :one-result
-                             :many-results))]
-        {:count filter-service-count})
-    " "
-    (tr [:service-search :total-services] {:total-service-count total-service-count})]
-   (doall
-    (for [result results]
-      ^{:key (::t-service/id result)}
-      [result-card e! app result]))
 
-   (if fetching-more?
-     [:span (tr [:service-search :fetching-more])]
-     (when (> filter-service-count (count results))
-       [scroll-sensor #(e! (ss/->FetchMore))]))])
+(defn results-listing [e! {service-search :service-search user :user :as app}]
+  (let [{:keys [results empty-filters? total-service-count
+                filter-service-count fetching-more?]} service-search
+        operator (:operator (:params app))]
+    [:div.col-xs-12.col-md-12.col-lg-12
+     [:p
+      (if operator
+        (tr (if (zero? filter-service-count)
+              [:service-search :operator-no-services]
+              [:service-search :operator-services])
+            {:name (::t-service/operator-name (first results))
+             :count filter-service-count})
+        (tr [:service-search (if empty-filters?
+                               :showing-latest-services
+                               (case filter-service-count
+                                 0 :no-results
+                                 1 :one-result
+                                 :many-results))]
+            {:count filter-service-count}))
+      " "
+      (tr [:service-search :total-services] {:total-service-count total-service-count})]
+     (doall
+      (for [result results]
+        ^{:key (::t-service/id result)}
+        [result-card e! (:admin? user) result]))
+
+     (if fetching-more?
+       [:span (tr [:service-search :fetching-more])]
+       (when (> filter-service-count (count results))
+         [scroll-sensor #(e! (ss/->FetchMore))]))]))
+
+(defn result-chips [e! chip-results]
+  (r/create-class
+    {:component-did-mount  ote.views.place-search/monkey-patch-chip-backspace
```

Comment:
You shouldn't need this monkey patching in company results, this was only needed because hand drawn geometries had a text field for changing the name.